### ISSUE TITLE
Add CardTrader prices

### ISSUE THREAD

**Issue by Emnalyeriar** [mtgjson/mtgjson#1209]
Hey, I asked about this on Discord ([link to message](https://discord.com/channels/224178957103136779/448295284137656342/1258192175611773031)) and was redirected here to make a feature request. As far as I know, there is data already pulled from Card Trader, I would like to suggest adding Card Trander prices to Price List :)

**Comment by ZeldaZach** [comment]
cc @kodawah Do we have some way to get this from an API or anything?

**Comment by kodawah** [comment]
yup, it's all available here https://www.cardtrader.com/docs/api/full/reference#marketplace-products

the trick is just loading the blueprint first, retrieving the scryfallid:cardId mapping, and then pull products by expansion. The main difference from tcg is that the api returns the whole dataset, so you need to filter until you find the first nm price
